libs/vkd3d/state.c

```diff
@@ -3287,6 +3308,11 @@ static bool d3d12_pipeline_state_can_use_dynamic_stride(struct d3d12_pipeline_st
                   graphics->minimum_vertex_buffer_dynamic_stride[slot]);
             return false;
         }
+        else if (dyn_state->vertex_strides[slot] & graphics->vertex_buffer_stride_align_mask[slot])
```

Comment:
Would we not erroneously hit this case if e.g. `vertex_buffer_stride_align_mask` is 4 and stride is 12?

Reply:
nvm, we always hit `graphics->vertex_buffer_stride_align_mask[i]--;` for every vertex buffer.